tests/kernel/update_kernel.pm

```diff
@@ -315,6 +333,9 @@ sub run {
         kgraft_state;
     }
     elsif (get_var('AZURE')) {
+        # When testing the first kernel-azure package in a new SLE release,
+        # uncomment the line below and comment out the other two.
+        #first_azure_release($repo);
```

Comment:
Wouldn't be variable better ? You would just  pre-add or clone the job with, e.g. FIRST_AZURE variable, instead of (un)commenting.

Reply:
Good idea. Done.